package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1673,29 +1678,99 @@ class Bonds(_Connection):
     transplants = defaultdict(list)
 
     def bonded_atoms(self):
-        """An AtomGroup of all atoms bonded to this Atom"""
+        """An :class:`~MDAnalysis.core.groups.AtomGroup` of all
+        :class:`Atoms<MDAnalysis.core.groups.Atom>` bonded to this
+        :class:`~MDAnalysis.core.groups.Atom`."""
         idx = [b.partner(self).index for b in self.bonds]
         return self.universe.atoms[idx]
 
     transplants[Atom].append(
         ('bonded_atoms', property(bonded_atoms, None, None,
                                   bonded_atoms.__doc__)))
 
+    def fragnum(self):
```

Comment:
not a huge fan of this, but I see why it's needed

maybe fragindices to denote it as something we've defined, in line with resindices

Reply:
Sure, no problem. Just out of interest: why aren't 'molnums' called 'molindices'?

Reply:
molnum is directly read from a tpr file, ie it's gromacs' opinion of molecules

Reply:
Thx, now that makes sense.